doc/using-js-ipfs-in-the-browser.md

```diff
@@ -0,0 +1,69 @@
+Using JS IPFS in The Browser
+----------------------------
+
+JS IPFS is the implementation of IPFS protocol in JavaScript. It can run on any
+evergreen browser, inside a service or web worker, as a web extention and in Node.js.
+
+The documentation details more information about running JS IPFS in the browser and
+boosting the speed with reducing the errors you face often because of limitation of
+js-ipfs.
+
+There are few different components for implementing JS IPFS in the browser. These
+are webrtc and websocket-star which will make it easier for implemented JS IPFS
+in the browser and boost the speed and reduce the errors.
+
+You can find the information present for enabling webRTC support for js-ipfs in
+the browser [here](https://github.com/ipfs/js-ipfs#how-to-enable-webrtc-support-for-js-ipfs-in-the-browser) and a different one is
+implementing with webrtc-star, you can find information [here](https://github.com/ipfs/js-ipfs#is-there-a-more-stable-alternative-to-webrtc-star-that-offers-a-similar-functionality).
+
+You can find information about running IPFS in the broswer [here](https://github.com/ipfs/js-ipfs#table-of-contents).
+
+Getting Data From IPFS
+-----------------------
+
+Using Javascript in the browser
+
+```js
+const node = new IPFS()
+
+node.once('ready', () => {
+  node.pipe('QmPChd2hVbrJ6bfo3WBcTW4iZnpHm8TEzWkLHmLpXhF68A', (err, data) => {
+    if (err) return console.error(err)
+
+    // convert Buffer back to string
+    console.log(data.toString())
+  })
+})
+```
+
+JS IPFS Restriction in Web Browser
+------------------------------------------
+
+- DHT in js-ipfs is not enabled by default.
+
+  In browser a JS IPFS node is more restricted because of the same origin policy
+  that decrease the discover/connect with peers, external peers without rendezvous
+  and relay servers, delegating peer/content routing and preload servers for content
+  discoverablilty hard and erroneous even with swarm peers.
```

Comment:
Let's rephrase this section to focus on discovery and connectivity, something like:

> This means js-ipfs running in the browser is limited to Web APIs available on a web page.
> There is no raw TCP sockets nor low level UDP, only WebRTC and Websockets.
> Discovery and connectivity to other peers is achieved with a mix of rendezvous and relay servers,
> delegated peer/content routing and preload servers.